netbox/circuits/models.py

```diff
@@ -300,6 +300,12 @@ class CircuitTermination(CableTermination):
         blank=True
     )
 
+    # Paths do not end on cable terminations, they continue at the other end of the circuit
+    is_path_endpoint = False
```

Comment:
Let's avoid setting arbitrary flags on model classes. The values aren't going to change among instances, so there's no need to store it on the class.

Instead, we can create e.g. `PATH_ENDPOINT_MODELS` and `CONNECTION_ENDPOINT_MODELS` constants, and check for `type(instance) in constants.PATH_ENDPOINT_MODELS`.